api/ratelimit/config/ratelimit/v3/rls_conf.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package ratelimit.config.ratelimit.v3;
+
+option java_package = "io.envoyproxy.ratelimit.config.ratelimit.v3";
+option java_outer_classname = "RlsConfigProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/ratelimit/config/ratelimit/v3;ratelimitv3";
+
+// [#protodoc-title: Rate limit service configuration]
```

Comment:
Can you add more comments on how this is going to be used with ADS? I assume the stream will be a series of RateLimitConfig either in SoTW or delta form?

Reply:
Added more comments with https://github.com/envoyproxy/ratelimit/pull/371/commits/d7b55467c7d1d7dc89d1bc540e4b79b37884631a